### ISSUE TITLE
Why is the padded area of the expanded mask filled with white (masked) pixels during forced resize/context expansion?

### ISSUE THREAD

**Issue by jroessler** [lquesada/ComfyUI-Inpaint-CropAndStitch#119]
When forcing a resize or expanding the crop/context in the Inpaint Crop node, the following line in crop_magic_im fills the expanded_mask with ones (i.e., white/masked):

`expanded_mask = torch.ones((mask.shape[0], expanded_image_h, expanded_image_w), device=mask.device)`

Reference:
https://github.com/lquesada/ComfyUI-Inpaint-CropAndStitch/blob/b432b2411cbb7d3192d35953bd3aafea05a0e245/inpaint_cropandstitch.py#L396

This causes the padded areas outside the original image/mask to be marked as masked (to be inpainted). During inpainting, this leads the model to modify these margins, which are just repeated edge pixels, resulting in potentially unwanted changes or artifacts at the crop boundary.

Wouldn't it be more intuitive to fill the expanded mask with zeros (unmasked) instead, so only the original mask region is inpainted? For instance:

`expanded_mask = torch.zeros((mask.shape[0], expanded_image_h, expanded_image_w), device=mask.device)`

Is there a specific reason for the current behavior? Otherwise, I'd love to submit a PR.

**Comment by jroessler** [comment]
Edit:

I just realized that the same issue occurs when using `extend_for_outpainting=True`. Specifically, in the extend_imm function, the following line extends the mask with white pixels (ones), so padded areas are also marked as masked:

`expanded_mask = torch.ones(1, new_H, new_W, device=mask.device)`

Reference:

https://github.com/lquesada/ComfyUI-Inpaint-CropAndStitch/blob/b432b2411cbb7d3192d35953bd3aafea05a0e245/inpaint_cropandstitch.py#L166C5-L166C68

This leads to the same effect as with forced cropping: extended mask regions outside the original image are set to white and will be inpainted.

**Comment by lquesada** [comment]
If they were filled in black, extend_for_outpainting wouldn't trigger outpainting.

With regards to the first question - filling them in white actually makes the inpainting work better. If they were filled with black mask, the model wouldn't replace them and the inpainting would be influenced by the edges out of the image.

Note that this all happens because you're enforcing an aspect ratio and an area that (both) don't fit in the image, hence the need to go out of it.
